index.bs

```diff
@@ -5783,10 +5779,12 @@ The {{MLOperand}} objects are created by the methods of {{MLGraphBuilder}}, inte
 
 <details open algorithm>
   <summary>
-    To <dfn>check dimensions</dfn> given [=/list=] |dimensions| and {{MLOperandDataType}} |type|, run the following steps:
+    To <dfn>check dimensions</dfn> given {{MLOperandDescriptor}} |descriptor|, run the following steps:
```

Comment:
I appreciate bundling of both params into a single descriptor, but I wonder if it was that way before for cases like with https://www.w3.org/TR/webnn/#api-mlgraphbuilder-expand which has a separate `newShape` parameter (but not a full descriptor since the type remains the same as the input). Granted, `expand` doesn't currently call "check dimensions" anyway, instead relying on the bidirectional broadcasting, but then I'm not seeing the broadcasting function call "check dimensions" either. So somewhere along the chain `expand` should call check dimensions. Though, that's not part of this CR, and we can always split it back if needed later. So signing off.

Reply:
Yeah, the reason for combining is that only a descriptor has a "byte length" so we'd need to rework that definition too if we kept it split.